tests/e2e/specs/in-game/ranked-matches.spec.js

```diff
@@ -0,0 +1,82 @@
+const dayjs = require('dayjs');
+import { setupGameAsP0, setupGameAsP1, Card } from '../../support/helpers';
+import { seasonFixtures } from '../../fixtures/statsFixtures';
+import { playerOne, playerTwo } from '../../fixtures/userFixtures';
+
+describe('Creating And Updating Ranked Matches', () => {
```

Comment:
I plan to move these tests into the gameOver.spec, which I think best captures the context of these tests anyway